deploy/examples/cluster-on-pvc-minikube.yaml

```
@@ -7,6 +7,11 @@
 # For example, to create the cluster:
 #   kubectl create -f crds.yaml -f common.yaml -f operator.yaml
 #   kubectl create -f cluster-on-pvc-minikube.yaml
+#
+# or by using the using the tests/scripts/create-dev-cluster.sh and the following minikube configuration:
+
```

Nit: I think this blank line could use a comment hash to keep consistency.

```suggestion
#
```